tests/test_utils/test_utils.py

```
@@ -0,0 +1,629 @@
+
+    Args:
+        dir_path (str): The path to the directory to remove.
+    """
+    if not dir_path:
+        raise ValueError("Directory path is empty or None.")
+
+    if not os.path.exists(dir_path):
+        return
+
+    dri_path = os.path.abspath(dir_path)
+    if not "tests/test_data" in dri_path:
```

Ruff (0.8.2)

Test for membership should be `not in`

Convert to `not in`

(E713)